.evergreen/config/test-task-groups.yml

```diff
@@ -0,0 +1,37 @@
+task_groups:
+  - name: test_atlas_task_group
+    setup_group:
+      - func: "fetch source"
+      - func: "prepare resources"
+      - func: "fix absolute paths"
+      - func: "make files executable"
+      - func: "install dependencies"
+      - func: "locate PHP binaries"
+      - func: "fetch extension"
+      - func: "install composer"
+      - command: subprocess.exec
+        params:
+          working_dir: src
+          binary: bash
+          add_expansions_to_env: true
+          env:
+            MONGODB_VERSION: '7.0'
+          args:
+            - ${DRIVERS_TOOLS}/.evergreen/atlas/setup-atlas-cluster.sh
```

Comment:
I noted this defaults `MONGODB_VERSION` to 6.0. Presumably 7.0 is required because this task group is used for search index testing (see: [Setup](https://github.com/mongodb/specifications/tree/master/source/index-management/tests#setup)).